README.asciidoc

```diff
@@ -1,16 +1,41 @@
-RPMGrill User Guide
-===================
+RPMGrill
+========
 :doctype: book
 :toc:
 :lang: en
 
+Overview
+--------
+
```

Comment:
You sure? Always felt those tags were a bit corny, especially once the builds stop passing ;)